challenge-3/src/utils.ts

```diff
@@ -0,0 +1,50 @@
+import { Interface } from "@ethersproject/abi";
+import { BigNumber, Contract, providers } from "ethers";
+import { Alert, AlertQueryOptions, AlertsResponse, Finding, GetAlerts } from "forta-agent";
+import { CHAIN_IDS } from "./constants";
+import * as fs from "fs";
+
+export type Addresses = {
+  l1Dai: string;
+  l2Dai: string;
+  escrowArbitrum: string;
+  escrowOptimism: string;
+};
+
+export const getEscrowBalances = async (
+  iface: Interface,
+  provider: providers.Provider,
+  blockNumber: number,
+  l1DaiAddress: string,
+  escrowArbitrumAddress: string,
+  escrowOptimismAddress: string
+): Promise<any> => {
```

Comment:
I think it is a good practice to stay away from the use of 'any', you could create a type for returning escrow balances.